php/php.editor/test/unit/data/testfiles/codegen/testIntersectionTypesImplementMethod03/testIntersectionTypesImplementMethod03.php.testIntersectionTypesImplementMethod03.codegen

```diff
@@ -1,2 +1,2 @@
-public function testMethod(\Test1\Foo&\Test1\Bar $param): \Test1\Foo&\Test1\Bar{
+public function testMethod(Foo&Bar $param): \Test1\Foo&\Test1\Bar{
```

Comment:
Is it expected that there is no namespace present anymore?

Reply:
https://github.com/apache/netbeans/blob/4086bd3bd687c616694d74f21aeab2ff6addc329/php/php.editor/test/unit/data/testfiles/codegen/testIntersectionTypesImplementMethod03/testIntersectionTypesImplementMethod03.php#L21-L42

Ideally, we should add 
```php
public function testMethod(Foo&Bar $param): Foo&Bar {
}
```
then, add use statements. Maybe, users expect that, I think...

Users have to fix uses if we don't add namespaces like this. Should we add FQNames like so far? (Of course, we also have to fix return types.)

Reply:
I don't know :sweat_smile:  Well, let's try this and wait for a possible bug fix? I mean, personally, I prefer uses + short names, so this is OK for me (yes, maybe use statements should be added, but this can be done later, as another ticket).

Reply:
> let's try this and wait for a possible bug fix?

Yes :)

Reply:
@tmysik I've reconsidered this... perhaps the user will run "Fix imports..." with or without namespaces.
I'll revert this to the previous behavior because it works fine if namespaces exist.

Reply:
Reverted https://github.com/apache/netbeans/compare/0a8820006d010a5fd1226b38de89c2c6fdaf4843..24794456b5a40ea57dadcefb5561aec732407d7f

Reply:
OK, sounds good 👍